core/src/Instance/B2CAuthority.cs

```diff
@@ -99,11 +103,7 @@ private bool InTrustedHostList(string host)
 
         protected override string GetDefaultOpenIdConfigurationEndpoint()
         {
-            Uri b2cAuthority = new Uri(CanonicalAuthority);
-
-            // To support host we would need to make the call to the tenant discovery end point, 
-            // which is not supported for b2clogin.com
-            string authorityUri = string.Format(CultureInfo.InvariantCulture, MicrosoftOnline + b2cAuthority.AbsolutePath + OpenIdConfigurationEndpoint);
+            string authorityUri = string.Format(CultureInfo.InvariantCulture, new Uri(CanonicalAuthority) + OpenIdConfigurationEndpoint);
```

Comment:
>authorityUri [](start = 19, length = 12)

nit: not sure it is worth it as this may only be used once.... it may be worth caching for this instance of the class.